### PR TITLE
[DSA Array 1.1.2 Task 1]: Reverse an Array 

### LINKED ISSUE #110: DSA 1.1.2 Arrangement, Rearrangement in Arrays (Reversal, rerrange zeros/+ve/-ve)
## Description
Standard rearrangement problems like reversing, rearrange positive and negative, and move zeros to end.

## Task
- [ ] Documentation
- [ ] Audio
- [ ] Video

## Note
- While requesting the issue to be assigned, please mention your **NAME & BATCH NUMBER**.
- Kindly refer to [Issue Guidelines](https://github.com/girlscript/winter-of-contributing/blob/DSA/DSA/CONTRIBUTING.md#issues).
- In case you see that the issue has already been assigned, try to request for a different task within the same issue.
- Changes should be made inside the ````DSA/directory```` & ````DSA```` branch.
- Name of the file should be same as the issue.
- This issue is only for 'GWOC' contributors of 'DSA' Domain.

### PR DESCRIPTION
**Pull request by NiharikaSngh**
<hr>

## Description 📜
I have made a documentation for the Task 1 of this issue.
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #110  

<hr>
## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [x] DSA
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Review by NiharikaSngh — COMMENTED** [review]
The mentor is requested to merge only the latest commit named Create 1.1.2 Task 1 into the GWOC DSA directory.

**Review by Muskan-creator — CHANGES_REQUESTED** [review]
Use markdown to add code snippets ,organise the code Write problem statement first.
refer this to add:https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet

**Comment by NiharikaSngh** [comment]
@Muskan-creator How to convert to draft? Yes, this is still in progress and 2 more tasks to be completed. I know markdown and have used it. Besides, the problem statement is specified in the heading itself. I have given code output as a hyperlink as the image is bigger. Code is already organised. Plz merge my PR !

File: DSA/1.1 Arrays/1.1.2 Task 1 #110.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+		end--;  }  
+}  	
+
+/* Utility function to print an array */  
+void printArray(int arr[], int size)    
+{  
+   for (int i = 0; i < size; i++)  
+   cout << arr[i] << " ";  
+
+   //cout << endl;
```
Comment (get-satyam) [inline DSA/1.1 Arrays/1.1.2 Task 1 #110.md:42]: Pls resolve this error at line 42

**Pull request merged**